### ISSUE TITLE
Bridging Drupal (special) versioning <-> semantic versioning

### ISSUE THREAD

**Issue by bassettsj** [drupalprojects/kalatheme#7]
Still trying to figure out how to match the proprietary versioning used in Drupal land and the semantic versioning used by other communities. I would like to open this issue to figure out a plan for version planning and keeping the two version numbers in sync.

**Comment by andrewmallis** [comment]
semserver: http://semver.org was pointed to a while back in the community. I've worked on some projects pre-kalamuna where we tried implementing it in features namespacing, but it got a bit confusing.

Here is the big debate: https://drupal.org/node/1612910 with lots of meaty nuggets.

A bridge approach could work well, but could also introduce complexity that poses a barrier to participation from members of the d.o community if they are trying to track back and post against an issue on a d.o release that does not appear in the project history on github.
